test/server/server_fuzz_test.cc

```diff
@@ -55,9 +55,11 @@ makeHermeticPathsAndPorts(Fuzz::PerTestEnvironment& test_env,
   }
   for (auto& cluster : *output.mutable_static_resources()->mutable_clusters()) {
     for (auto& health_check : *cluster.mutable_health_checks()) {
-      // TODO(asraa): QUIC is not enabled in production code yet, so remove references for HTTP3.
+      // QUIC is not enabled in production code yet, so remove references for HTTP3.
```

Comment:
very minor nit: could you leave the TODOs? (TODO(asraa), or TODO(zasweq)), since they're good for catching this with a grep or a search?

Reply:
Wait, but how is it a todo if its already fixed :)?

Reply:
Oh, because the TODO is alerting that when the issue is fixed, we can remove the manual clearing of the field.

Reply:
Ohh gotcha, also the CI for this is really weird...it says the sainitizedInput is never modified in this class...but it clearly is with clear_codec_client_type?

Reply:
Im just gonna leave your name as the todo, as this might be added to production after my internship!

Reply:
Hopefully I'll be back though haha

Reply:
Could you merge main in? It looks like clang_tidy is running with gcc compiler (which shouldn't be happening, was fixed last week)

Reply:
Will merge